pyface/workbench/action/view_menu_manager.py

```diff
@@ -88,10 +88,7 @@ def _groups_default(self):
     # 'ViewMenuManager' interface.
     # ------------------------------------------------------------------------
 
-    @on_trait_change(
-        "window.active_perspective,window.active_part,"
-        "window.views,window.views_items"
-    )
+    @observe("window.active_perspective,window.active_part,window.views.items")
     def refresh(self):
```

Comment:
This is a bug and not just a style recommendation.

```suggestion
    def refresh(self, event):
```

Reply:
I've made this mistake myself on a number of occasions, and I think it's going to be a very easy one for projects to make when converting from `on_trait_change` to `observe`. This is something it may be worth catching in Traits itself. I've opened enthought/traits#1428.